facebook/__init__.py

```diff
@@ -362,6 +364,18 @@ def debug_access_token(self, token, app_id, app_secret):
         }
         return self.request(self.version + "/" + "debug_token", args=args)
 
+    def get_auth_url(self, app_id, canvas_url, perms=None, **kwargs):
+        """
+        Build link to make login though Facebook Open Authorization (OAuth).
+        """
+        url = FACEBOOK_URL + self.version + '/' + FACEBOOK_OAUTH_DIALOG_URI
```

Comment:
Can we use [`.format`](https://pyformat.info/) instead of string concatenation? See #326.